copy/copy.go

```diff
@@ -126,6 +127,16 @@ type Options struct {
 	// Download layer contents with "nondistributable" media types ("foreign" layers) and translate the layer media type
 	// to not indicate "nondistributable".
 	DownloadForeignLayers bool
+
+	// Contains slice of OptionCompressionVariant, where c/image will ensure that selected
+	// images in the manifest list will exist with compression specificed in OptionCompressionVariant
+	// while performing copy.
```

Comment:
- “c/image” is not something callers really care about here.
- if “selected images” refers to `ImageListSelection`, that’s probably worth being more explicit
- Please add “Invalid when copying a non-multi-architecture image. That will probably change in the future”. (There’s another review comment outstanding about actually rejecting the option in that case.)